docs/tutorials/lcevc.md

```diff
@@ -0,0 +1,118 @@
+# LCEVC Quick Start 
+ 
+#### Requirements 
+ 
+ - Only supported on browsers with Media Source Extensions SourceBuffer support 
+ 
+ - MPEG-5 Part2 LCEVC decoding support (decoding provided by [lcevc_dec.js][], 
+ must be separately included) 
+ 
+##### Integration documentation : [docs](../design/lcevc-integration.md) 
+ 
+##### More on [MPEG-5 Part2 LCEVC][] 
+ 
+[lcevc_dec.js]: https://www.npmjs.com/package/lcevc_dec.js?activeTab=readme
+[MPEG-5 Part2 LCEVC]: https://www.lcevc.org 
+ 
+#### Configure LCEVC 
+ 
+LCEVC Decoder library needs to be included in the HTML page: 
+ 
+- Local `npm install lcevc_dec.js` 
+```html 
+<script src="../node_modules/lcevc_dec.js/dist/lcevc_dec.min.js"></script> 
+``` 
+- Or use CDN.js for latest LCEVC Decoder libraries. 
+```html 
+<script src="https://unpkg.com/lcevc_dec.js@1.0.0/dist/lcevc_dec.min.js"></script> 
+``` 
+ 
+Configuration to enable LCEVC enhancement: 
+```js 
+player.configure('lcevc.enabled', true); 
+``` 
+ 
+#### LCEVC setup via UI library 
+ 
+Sample setup using the Shaka Player UI library:  
+ 
+```html 
+<!DOCTYPE html> 
+<html> 
+  <head> 
+    <!-- Shaka Player UI compiled library and default CSS for player controls: --> 
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/shaka-player/4.3.5/shaka-player.ui.debug.min.js"></script> 
```

Comment:
This couldn't possibly work as written, since Shaka v4.3.5 doesn't support lcevc_dec.

Reply:
Moved this to use Shaka Player UI library generated in the dist locally.